tests/regression/roles/sonic_port_group/defaults/main.yml

```diff
@@ -37,7 +37,7 @@ tests:
     description: Set all port group to default speeds
```

Comment:
It is just one port group, now, so please change this line to something like:

```suggestion
    description: Set a port group to the default speed.
```

Reply:
Done. Thanks!

Reply:
Thank you.

Resolved.